Source/PokemonRNGSystem/BaseRNGSystem.cpp

```diff
@@ -170,13 +170,14 @@ BaseRNGSystem::predictStartersForNbrSeconds(u32 seed, const int nbrSeconds)
 }
 
 BaseRNGSystem::SecondaryCandidate BaseRNGSystem::generateSecondary(u32 seed, const Stats baseStats,
-                                                                   const int level,
-                                                                   const u8 genderRatio)
+                                                                   const int level, const u8 genderRatio, 
+																   int frameNumber)
```

Comment:
formatting, pls run clang-format (you can use the vs plugin here: https://marketplace.visualstudio.com/items?itemName=LLVMExtensions.ClangFormat&ssr=false )